date_range/README.rst

```diff
@@ -41,6 +41,8 @@ Installation
 
 The addon use the daterange method from postgres. This method is supported as of postgresql 9.2
 
+The `odoo_test_helper` Python library is needed for the tests.
```

Comment:
README is a user documentation, so this shouldn't be here.

Reply:
Thanks, I reverted the change :+1: